### ISSUE TITLE
Setting custom delimiters doesn't work with new highlighting

### ISSUE THREAD

**Issue by peteruhnak** [antlr/jetbrains-plugin-st4#28]
Previously for group files it was possible to declare custom delimiters and the highlighter acted accordingly.

```
delimiters "%", "%"

base(a, b) ::= <<
<root>
  <first>%a%</first>
  <second>%b%</second>
</root>
>>
```

Now it always highlights only `<` and `>`

Looking at the old code, this was removed:

https://github.com/antlr/jetbrains-plugin-st4/blob/5740453e43200dd333e82f08a0ea9412c0e8cdf5/src/main/java/org/antlr/jetbrains/st4plugin/highlight/STGroupSyntaxHighlighter.java#L71-L75

https://github.com/antlr/jetbrains-plugin-st4/blob/0c2abe124ea2a3c3e596c24923093ffb27ca1eb6/src/main/java/org/antlr/jetbrains/st4plugin/highlight/STSyntaxHighlighter.java#L44-L48

**Comment by bjansen** [comment]
Yeah, sorry about that, I completely missed this part in my refactoring :). Will fix it soon, thanks for reporting the bug!

**Comment by bjansen** [comment]
Could you please try this version and let me know if it works correctly for you? It recognizes custom delimiters when opening an existing file and when modifying the current editor, so I don't think I missed any test case.

[jetbrains-plugin-st4-0.9-SNAPSHOT.zip](https://github.com/antlr/jetbrains-plugin-st4/files/4833413/jetbrains-plugin-st4-0.9-SNAPSHOT.zip)

See this page for how to install plugins from disk: https://www.jetbrains.com/help/idea/managing-plugins.html#install_plugin_from_disk

**Comment by peteruhnak** [comment]
I can confirm that this fixed the issue. Thanks for the quick response & fix!

**Comment by bjansen** [comment]
Fixed in 7832eeba

**Comment by parrt** [comment]
@bjansen  are you still able to work on the plug-ins? thanks!
